bundle/manifests/backstage-operator.clusterserviceversion.yaml

```diff
@@ -206,7 +206,7 @@ spec:
                   value: quay.io/fedora/postgresql-15:latest
                 - name: RELATED_IMAGE_backstage
                   value: quay.io/janus-idp/backstage-showcase:next
-                image: quay.io/janus-idp/operator:0.0.1
+                image: gazarenkov/backstage-operator:latest
```

Comment:
this seems like an accidentally pushed change.

Reply:
yes it is

Reply:
fixed